samples/VK.gs

```diff
@@ -0,0 +1,81 @@
+/**
+ *
+ * VK's Auth flow https://vk.com/dev/authcode_flow_user
+ * Scopes list https://vk.com/dev/permissions
+ * Be careful. Implicit Flow's Methods are not used on the server side. They will always return "error_code": 15, "error_msg": "Access denied: no access to call this method"
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+/**
+ * Authorizes and makes a request to the VK API.
+ */
+function run() {
+  var service = getService();
+  if (service.hasAccess()) {
+    // GET requests require access_token parameter
+    var url = 'https://api.vk.com/method/groups.get?access_token=' + service.getAccessToken();
+    var response = UrlFetchApp.fetch(url);
+    var result = JSON.parse(response.getContentText());
+    Logger.log(JSON.stringify(result, null, 2));
+  } else {
+    var authorizationUrl = service.getAuthorizationUrl();
+    Logger.log('Open the following URL and re-run the script: %s',
+        authorizationUrl);
+  }
+}
+
+/**
+ * Reset the authorization state, so that it can be re-tested.
+ */
+function reset() {
+  var service = getService();
+  service.reset();
+}
+
+/**
+ * Configures the service.
+ */
+function getService() {
+  return OAuth2.createService('VK')
+      // Set the endpoint URLs.
+      .setAuthorizationBaseUrl('https://oauth.vk.com/authorize')
+      .setTokenUrl('https://oauth.vk.com/access_token')
+
+      // Set the client ID and secret.
+      .setClientId(CLIENT_ID)
+      .setClientSecret(CLIENT_SECRET)
+
+      // Set the name of the callback function that should be invoked to complete
+      // the OAuth flow.
+      .setCallbackFunction('authCallback')
+
+      // Set the property store where authorized tokens should be persisted.
+      .setPropertyStore(PropertiesService.getUserProperties())
+
+      // Set the scope and additional specific parameters if its are supported
+      .setScope('groups,offline')
+      .setParam('approval_prompt', 'force');
```

Comment:
This doesn't seem to be a supported parameter.

Reply:
@erickoledadevrel , my fail! Thanks God I'm not in Google! =)